src/components/TopNav/hooks.ts

```diff
@@ -27,3 +28,18 @@ export function useNavigation() {
 
   return { currentPath, isActive };
 }
+export function findServiceByPath(servicePath: string): NavLink | null {
```

Comment:
I think this should throw an error when the nav isn't found.

```suggestion
export function findServiceByPath(servicePath: string): NavLink {
```